### PR TITLE
Contentful tests

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
Start implementing tests for the `lib/contentful` module.

#### How should this be reviewed?
- clone Gambit
- Run:
  - `npm install`
  - `npm run all-tests`

#### Checklist
- [x] Documentation added for new features/changed endpoints.

### CODE REVIEW

**Comment by rapala61** [comment]
Ugh, it seems it will still require merging develop into this branch, which will start the release-commit-ception thingy.

File: lib/contentful.js [hunk 1]
```diff
@@ -6,15 +6,17 @@
...
 const UnprocessibleEntityError = require('../app/exceptions/UnprocessibleEntityError');
 
-const defaultCampaignId = process.env.CONTENTFUL_DEFAULT_CAMPAIGN_ID || 'default';
+const defaultCampaignId = config.defaultCampaignId;
 
 // The sequence we define peroperties here determines the order they appear in GET Gambit Campaigns
 // API response. Should match the sequence defined in Contentful, which is based on the order in
 // which our end user will see the messages.
+// TODO: Javascript Objects do not guarantee order,
+// we should change this to an array if order is important.
```
Comment (aaronschachter) [inline lib/contentful.js]: The order is only really used by the Slothbot "View Messages" functionality -- so order isn't critical. I think it's safe to leave as an object for now, as it's an easy way to map the machine name style variables with the corresponding Contentful field name.

File: test/lib/contentful.test.js [hunk 2]
```diff
@@ -1,8 +1,82 @@
 'use strict';
 
 require('dotenv').config();
-// const contentful = require('../../lib/contentful');
+const Promise = require('bluebird');
 const test = require('ava');
-// const chai = require('chai');
+const chai = require('chai');
+// const expect = require('chai').expect;
```
Comment (aaronschachter) [inline test/lib/contentful.test.js]: Should we remove this line?

Comment (rapala61) [inline test/lib/contentful.test.js]: Yea... I'll remove. 🎺 womp womp

**Pull request merged**